ui/components/app/edit-gas-fee-popover/network-status/network-status.js

```diff
@@ -32,7 +24,8 @@ const NetworkStatus = () => {
       <div className="network-status__info">
         <div className="network-status__info__field">
           <span className="network-status__info__field-data">
-            {estBaseFee !== null && `${estBaseFee} GWEI`}
+            {gasFeeEstimates?.estimatedBaseFee &&
+              `${gasFeeEstimates?.estimatedBaseFee} GWEI`}
           </span>
```

Comment:
@mcmire : this is how it looks now with wrapping:
<img width="320" alt="Screenshot 2021-11-24 at 12 28 20 AM" src="https://user-images.githubusercontent.com/2182307/143087004-9683aa4f-0d80-4246-8bc4-97986181a289.png">

Reply:
Okay. That's fine for now. If this gets annoying we can tweak it some more but considering this really only happens on testnets then I'm good with this 👍🏻